test/evergreen_develop.yml

```diff
@@ -3,15 +3,12 @@
 include:
 - filename: test/evergreen.yml
 
-
-# Build variants that need to be run only on the develop branch.
-buildvariants:
-- name: ubuntu2004-perf-tests
-  display_name: Ubuntu 20.04 Performance tests
+variables:
+# Template for perf-tasks
+- &perf-tasks-template
+  name: perf-tasks-template
```

Comment:
nit: I saw some discussion earlier that we had to keep name, but now the template has been moved to `variables` is it possible to drop the field?

Reply:
Yes, updated in [14c2725](https://github.com/wiredtiger/wiredtiger/pull/8909/commits/14c27259f64e056ac59eeb36ea4d77b0c9e571cc)